pkg/controller/deployment/rollback.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/klog/v2"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I have seen this flip between having and not having the alias depending on the IDE you use.

Is there any guidance that indicates that one is better than the other?

Reply:
> Is there any guidance that indicates that one is better than the other

I noticed this when I was dealing with golangci-lint duplicate imports.

This seems to be goimports' preference.

I think that if we want to deal with these issues, without modifying all the source code, calling the goimports automated processing and detection script is also needed.So the current issue doesn't do enough.

I think it's simply a matter of style, not performance. So I'm open-minded about whether to modify.

```bash
$ go install golang.org/x/tools/cmd/goimports@latest
$ goimports -w -d pkg/controller/deployment/rollback.go
diff -u pkg/controller/deployment/rollback.go.orig pkg/controller/deployment/rollback.go
--- pkg/controller/deployment/rollback.go.orig	2022-08-16 22:03:52.000000000 +0800
+++ pkg/controller/deployment/rollback.go	2022-08-16 22:03:52.000000000 +0800
@@ -25,7 +25,7 @@
 	"k8s.io/klog/v2"

 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	extensions "k8s.io/api/extensions/v1beta1"
 	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 )
```

Reply:
maybe worth enabling the `goimports` linter in golangci-lint then, so that we can stay consistent.